docs/actions/motion.create_forwarded.md

```diff
@@ -25,6 +27,8 @@ The original motion must be updated as well (this is done by the automatic relat
 * The unique `id` of the newly created motion has to be linked to the _origin motion_s `derived_motion_ids` field.
   * Deleting the newly created motion has to ensure that the corresponding entry was removed from the _origin motion_s `derived_motion_ids` field
 
+The optional flags `use_original_submitter` and `use_original_number` will cause the original submitters and original numbers to be used in the new motion respectively. In case of the submitters, the action will use the origin meetings meeting_user for the `meeting_user_id` field of the new submitter instance in the target meeting.
```

Comment:
IMO it is no good idea to use the meeting_user from other meeting in target meeting.
Alternatives:

1. Create meeting_user in target_meeting with empty group (could be used also for committee user)
2. motion_submitter should use `user`  instead of `meeting_user`. Disadvantage: getting forward motions per meeting and user only via filter

@rrenkert should have a look too

Reply:
WDYM by empty group?

Reply:
IMO we shouldn't use meeting_user instances from meeting A in meeting B. The meeting_id is an integral part of an meeting_user instance. 
Empty group means a group without any permission. This could be part of a solution for alternative 1.

Reply:
@rrenkert what should happen here?

Reply:
outdated, this solution was removed